src/dash-table/derived/paginator.ts

```diff
@@ -33,6 +47,43 @@ function getBackEndPagination(
 
             page_current--;
             setProps({ page_current, ...clearSelection });
+        },
+        loadFirst: () => {
+            page_current = 0;
+            setProps({ page_current, ...clearSelection });
+        },
+        loadLast: () => {
+            if (page_count) {
+                page_current = page_count;
+                setProps({ page_current, ...clearSelection });
+            }
+        },
+        lastPage: page_count,
+        goToPage: (page: number) => {
+
+            // adjust for zero-indexing
+            page--;
+
+            page_current = page;
+
+            if (page < 0) {
+                page_current = 0;
+            }
+
+            if (page_count && page > page_count) {
+                page_current = page_count;
+            }
+
+            setProps({ page_current, ...clearSelection });
+        },
+        disablePrevious: () => {
+            return page_current === 0;
+        },
+        disableNext: () => {
+            return page_count !== undefined && page_current === page_count;
+        },
+        disableLast: () => {
+            return !page_count ? true : page_current === page_count;
```

Comment:
🌴`getFrontEndPagination` and `getBackEndPagination` seem to share a significant amount of logic